apps/prairielearn/src/tests/permissions/courseAdminAccess.test.ts

```diff
@@ -16,7 +18,16 @@ import * as helperServer from '../helperServer.js';
 
 const sql = sqldb.loadSqlEquiv(import.meta.url);
 
-async function checkPermissions(users) {
+interface User {
```

Comment:
Maybe we shouldn't re-use names from `db-types`, to avoid potential confusion when reading code?